website/src/documentation/programming-guide.md

```diff
@@ -590,6 +615,12 @@ words = ...
 %}
 {% github_sample /apache/beam/blob/master/sdks/python/apache_beam/examples/snippets/snippets_test.py tag:model_pardo_apply
 %}```
+```go
+func computeWordLengthFn(word string) int {
+      return len(word)
+}
+wordLengths := beam.ParDo(s, computeWordLengthFn, words)
```

Comment:
Consider more closely matching the java or python examples in structure.

Please add a blank line after the function definition.